native/src/index.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+'use strict';
+
+const { processRequest } = require('./parse');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
+});
+
+rl.on('line', l => console.log(processRequest(l)));
```

Comment:
:trollface: now it's the same in javascript's, so I get i don't have to changes this